Last-Update: 2015-04-03
Forwarded: no
Author: Dmitry Smirnov <onlyjob@member.fsf.org>
Description: change default date format to region-neutral ISO notation with time zone.

--- web/skins/classic/views/monitor.php
+++ web/skins/classic/views/monitor.php
@@ -69,9 +69,9 @@
         'Width' => "320",
         'Height' => "240",
         'Orientation' => "0",
         'Deinterlacing' => 0,
-        'LabelFormat' => '%N - %d/%m/%y %H:%M:%S',
+        'LabelFormat' => '%N - %Y-%m-%d %H:%M:%S %z',
         'LabelX' => 0,
         'LabelY' => 0,
         'ImageBufferCount' => 50,
         'WarmupCount' => 25,
